src/app/shared/components/hmcts-em-viewer-ui/viewers/annotation-pdf-viewer/rotation-toolbar/rotation.component.ts

```diff
@@ -9,14 +11,44 @@ import { EmLoggerService } from '../../../logging/em-logger.service';
     styleUrls: ['./rotation.component.scss'],
     providers: []
 })
-export class RotationComponent {
+export class RotationComponent implements OnInit, OnDestroy {
+    showRotationButton: boolean;
+    rotationButtonStatusSub: Subscription;
+
     @Input() pageNumber: number;
+    @ViewChild('rotationButton') rotationButton: ElementRef
 
     constructor(private pdfRenderService: PdfRenderService,
-                private log: EmLoggerService) {
+                private log: EmLoggerService,
+                private rotationService: RotationService,
+                private renderer: Renderer2) {
         this.log.setClass('RotationComponent');
     }
 
+    ngOnInit() {
```

Comment:
not sure what the logic is here. I would've assumed this is either shown or hidden by default, then when clicked it's toggled